### ISSUE TITLE
Weird NoRouteToHostException trying to run the tests

### ISSUE THREAD

**Issue by th0br0** [miguno/kafka-storm-starter#2]
Hey there, I'm getting this weird NoRouteToHostException when I try to run the tests (1.7.0_60). Any idea what might cause this?

```
> test

[error] Uncaught exception when running tests: java.net.NoRouteToHostException: No route to host
[trace] Stack trace suppressed: run last test:test for the full output.
Exception in thread "Thread-1" java.io.EOFException
    at java.io.ObjectInputStream$BlockDataInputStream.peekByte(ObjectInputStream.java:2598)
    at java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1318)
    at java.io.ObjectInputStream.readObject(ObjectInputStream.java:370)
    at sbt.React.react(ForkTests.scala:116)
    at sbt.ForkTests$$anonfun$mainTestTask$1$Acceptor$2$.run(ForkTests.scala:75)
    at java.lang.Thread.run(Thread.java:745)
```

**Comment by th0br0** [comment]
Nevermind, this seems scalatest related, see scalatest/scalatest#248

**Comment by robinloxley1** [comment]
The issue of scalatest was fixed in 2.2.4.
